src/styles/blocks/main.scss

```diff
@@ -0,0 +1,136 @@
+* {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
```

Comment:
You have already created styles for this selector in the normalize scss, put this styling here